datasafe-types-api/src/test/java/de/adorsys/datasafe/types/api/shared/AwsClientRetry.java

```diff
@@ -0,0 +1,30 @@
+package de.adorsys.datasafe.types.api.shared;
+
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.model.AmazonS3Exception;
+import lombok.SneakyThrows;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class AwsClientRetry {
+    @SneakyThrows
+    public static void createBucketWithRetry(AmazonS3 client, String bucket) {
```

Comment:
Due to semantics of this function and its usage suggest to name it `createBucketWithRetryForMinio`

Reply:
In this case it is used for minio. But it can be used for any S3Amazon Client. So I leave name as it is.